fenix/app/src/main/java/org/mozilla/fenix/compose/Chip.kt

```diff
@@ -79,20 +79,16 @@ fun SelectableChip(
     selectableChipColors: SelectableChipColors = SelectableChipColors.buildColors(),
     onClick: () -> Unit,
 ) {
-    var selected by remember { mutableStateOf(isSelected) }
-
-    Chip(
-        text = text,
-        backgroundColor = if (selected) {
-            selectableChipColors.selectedBackgroundColor
-        } else {
-            selectableChipColors.unselectedBackgroundColor
-        },
-        isSquare = isSquare,
-        textColor = if (selected) selectableChipColors.selectedTextColor else selectableChipColors.unselectedTextColor,
+    FilterChip(
+        onClick = onClick,
+        selected = isSelected,
+        shape = if (isSquare) RoundedCornerShape(4.dp) else RoundedCornerShape(25.dp),
+        colors = selectableChipColors.toMaterialChipColors(),
```

Comment:
I was initially hesitant to using something other than `Chip` here, but looking over the API for `FilterChip`, this seems like a better overall root component for this particular use case. 

my nit is just to re-order the parameters to match the ordering in the `FilterChip` declaration.